### PR TITLE
#363 roundup sftp drop

### LINKED ISSUE #363: Enhance weekly reporting to push to ADS SFTP server
## 💡 Description

Per discussions with ADS folks, they would prefer we deliver by SFTP. So let's add that as an additional delivery mechanism. SFTP information TBD

### PR DESCRIPTION
**Pull request by alexdunnjpl**
## 🗒️ Summary
Refactors roundup action and implements sftp-drop delivery for the roundup metadata

## ⚙️ Test Data and/or Report
unit tests pass

## ♻️ Related Issues
fixes #363

### CODE REVIEW

**Comment by nutjob4life** [comment]
Oh "roundup" … not [roundup-action](https://github.com/NASA-PDS/roundup-action).

**Comment by alexdunnjpl** [comment]
> Oh "roundup" … not [roundup-action](https://github.com/NASA-PDS/roundup-action).

yeeeeeeah... we can rename this if it's likely to confuse people more-than-rarely :sweat_smile:

File: src/pds_doi_service/core/actions/roundup/email.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+from pds_doi_service.core.actions.roundup.enumerate import get_previous_week_metadata
+from pds_doi_service.core.actions.roundup.metadata import RoundupMetadata
+from pds_doi_service.core.actions.roundup.output import prepare_doi_record
+from pds_doi_service.core.db.doi_database import DOIDataBase
+from pds_doi_service.core.entities.doi import DoiRecord
+from pds_doi_service.core.util.emailer import Emailer as PDSEmailer
+from pkg_resources import resource_filename
+
+
+def get_email_content_template(template_filename: str = "email_weekly_roundup.jinja2"):
```
Comment (nutjob4life) [inline src/pds_doi_service/core/actions/roundup/email.py:32]: Maybe add the return type hint `-> jinja2.Template`?

**Review by nutjob4life — APPROVED** [review]
👍

**Pull request merged**